packages/core/src/App/Containers/Layout/header/__tests__/traders-hub-header.spec.tsx

```diff
@@ -41,17 +35,12 @@ jest.mock('@deriv/hooks', () => ({
 }));
 
 describe('TradersHubHeader', () => {
-    const renderComponent = () =>
+    const mock_store = mockStore({
+        ui: { is_desktop: true },
+    });
+    const renderComponent = (modified_store = mock_store) =>
         render(
-            <StoreProvider
-                store={mockStore({
-                    feature_flags: {
-                        data: {
-                            next_wallet: true,
-                        },
```

Comment:
why do we remove this?

Reply:
in this test file, this was added by mistake, while copying from other test file, i suppose. so removing it now.
-4 lines 🤩